template/libtofs.jinja

```diff
@@ -87,12 +87,13 @@
         {%- else %}
           {%- set fs_dir = salt['config.get'](tplroot ~ ':tofs:dirs:default', 'default') %}
         {%- endif %}
-        {%- set url = '- salt://' ~ '/'.join([
+        {%- set url = [
+            '- salt:/',
             path_prefix_inc_ext,
             files_dir,
             fs_dir,
-            src_file.lstrip('/')
```

Comment:
I think we should keep the `.lstrip('/')` or if the user define the `src_file` as `/foo/bar.jinja`, you will have a double `/`

* `path_prefix_inc_ext = ''`
* `files_dir = ''`
* `fs_dir = ''`
* `src_file = '/foo/bar.jinja'`

Results in `- salt:///foo/bar.jinja`.

Reply:
We resolved this by discussing directly and agreeing that `strip` needs to be used for all of these, to deal with leading _and_ trailing slashes.